src/commands/suspend.ts

```diff
@@ -109,6 +110,22 @@ export const suspendCommand: CommandDef = {
       await user.send(
         'You have been suspended for violating our Code of Conduct. A channel has been created in the server for you to discuss this with the moderation team.'
       );
+      await userStatus.findOne({ userid: user.id }, function (
```

Comment:
don't use `function`, stick with leveraging `async/await`.

Also add a check/flag that goes off of if `MONGO_URI` is passed in the config, this way if someone didn't configure that property this feature still works, but just doesn't save anything to the db.

Reply:
Stuck it in an `if(config.MONGO_URI)` block. Changed to async, added await on the save and send.